realm/realm-library/build.gradle

```diff
@@ -30,6 +30,10 @@ if (!ext.coreArchiveDir) {
 ext.coreArchiveFile = rootProject.file("${ext.coreArchiveDir}/realm-sync-android-${project.coreVersion}.tar.gz")
 ext.coreDistributionDir = file("${projectDir}/distribution/realm-core/")
 ext.coreDir = file("${project.coreDistributionDir.getAbsolutePath()}/core-${project.coreVersion}")
+ext.ccachePath = project.hasProperty('ccachePath') ?
```

Comment:
This can be shortened to:
```
ext.ccachePath = project.findProperty('ccachePath') ?: System.getenv('REALM_CCACHE_PATH')
```
It's also safer in case of refactoring